gps2val/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright Robert Bosch GmbH, 2020. Part of the Eclipse Kuksa Project.
+# Copyright Robert Bosch GmbH, 2020-2023. Part of the Eclipse Kuksa Project.
```

Comment:
@SebastianSchildt - this license is quite different compared to the [DBC Docker file](https://github.com/eclipse/kuksa.val.feeders/blob/main/dbc2val/Dockerfile). Historical reasons or mistake?

Reply:
Historical reasons, EDL is basically public domain, and historically we have used it for some config or build scripts, where we thought "copyrighting" them is too ridiculous. I.e. I have no issues if somebody copies snippets and tricks out of our Dockerfiles without any attribution. 
But probably, to mak scanners happy it would be better to just add Apache header and hoping people have enough "criminal energy" to copy helpful stuff anyway